packages/vx-pattern/src/patterns/Circles.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import cx from 'classnames';
+import Pattern from './Pattern';
+
+export type PatternCirclesProps = {
+  /** Unique id for the pattern. */
+  id: string;
+  /** Width of the pattern element. */
+  width: number;
+  /** Height of the pattern element. */
+  height: number;
+  /** Radius of the pattern circles. */
+  radius?: number;
+  /** Fill applied to circles. */
+  fill?: string;
+  /** className applied to circles. */
+  className?: string;
```

Comment:
How are the props ordered by?

Reply:
there is not enforced ordering, I enabled several new rules in the `nimbus` upgrade but decided that was too big an undertaking / can't auto-fix.